docs/shared/core-tutorial/04-build-affected-projects.md

```diff
@@ -0,0 +1,91 @@
+# Core Nx Tutorial - Step 4: Build Affected Projects
+
+Nx scales your development by doing code change analysis to see what apps or libraries are affected by a particular pull request.
+
+**Commit all the changes in the repo**:
+
+```bash
+git add .
+git commit -am 'init'
+git checkout -b testbranch
+```
+
+**Open `packages/cli/src/ascii.go` and change the go code:**
+
+```go
+package main
+
+import (
+  "fmt"
+  "os"
+)
+
+func check(e error) {
+  if e != nil {
+      panic(e)
+  }
+}
+
+func main() {
+    fmt.Println("Hello, Dish of the Day")
+    dat, err := os.ReadFile("../ascii/assets/cow.txt")
+    check(err)
+    fmt.Print(string(dat))
+}
+```
+
+**Run `nx affected:apps`**, and you should see `cli` printed out. The `affected:apps` looks at what you have changed and uses the project graph to figure out which apps are affected by this change.
```

Comment:
Could you replace it with "nx print-affected --select=projects"? "affected:apps" and "affected:libs" should be deprecated